### ISSUE TITLE
Vidplay not working

### ISSUE THREAD

**Issue by NotTsuu** [mbebe/blomqvist#1055]
Vidplay hasn't been working for like two weeks, maybe more.

**Comment by mbebe** [comment]
fmovies fixed. Please update.

**Comment by NotTsuu** [comment]
Not working again, I feel bad for mbebe

**Comment by mbebe** [comment]
Ok. Updated.

**Comment by pitsi** [comment]
Thank you for fixing fmovies once more :) As it seems, all its issues have to do with decryption, so how can a simple user know that the addon fails because of that? 
Because if I knew that decryption was to blame, I would help finding a new provider for keys.

**Comment by mbebe** [comment]
I have a new script to get keys, so hopefully it will work some days or weeks maybe.

**Comment by pitsi** [comment]
Ok. But how can someone tell if the issue is the decryption or not?

**Comment by NotTsuu** [comment]
Hey guys! Vidplay has unfortunately stopped working again 😕

**Comment by pitsi** [comment]
Mycloud has stopped too. As it seems, the new keys lasted for roughly a week and the errors are the same.

I am tired of this cat and mouse game :(

**Comment by NotTsuu** [comment]
> Mycloud has stopped too. As it seems, the new keys lasted for roughly a week and the errors are the same.
> 
> I am tired of this cat and mouse game :(

I bet it's because they are finally starting to crack down on all the piracy.

Also it makes sense now cause ai is a thing now and they are probably using it to shut down every website and addon such and such.

**Comment by pitsi** [comment]
I wanted to write so much and explain why you are wrong, but I am so tired...

In other news, the keys used by the addon were updated a few hours ago and now mycloud and vidplay work again.

**Comment by NotTsuu** [comment]
> I wanted to write so much and explain why you are wrong, but I am so tired...
> 
> In other news, the keys used by the addon were updated a few hours ago and now mycloud and vidplay work again.

Yeah I checked vidplay yesterday at midnight to see if it worked and it did so now it's fixed and hopefully it lasts longer than a week lol

**Comment by NotTsuu** [comment]
Anyone getting the playback error?

**Comment by pitsi** [comment]
No, vidplay and mycloud play fine here.

**Comment by NotTsuu** [comment]
> No, vidplay and mycloud play fine here.

Hmm, weird. When I'm trying to watch the walking dead  it ends up saying one or more items failed to play. I have checked the website to see if the episode plays and it works completely fine but for kodi, it doesn't work at all.

**Comment by pitsi** [comment]
I just tested the walking dead s11e01 and s11e24 (first and last available episode from the latest season) and they both played fine on both vidplay and mycloud. Screenshots upon request.

**Comment by NotTsuu** [comment]
> I just tested the walking dead s11e01 and s11e24 (first and last available episode from the latest season) and they both played fine on both vidplay and mycloud. Screenshots upon request.

It seems that vidplay is now broken once again and mycloud doesn't show anymore. Keys might need updating again.

**Comment by pitsi** [comment]
It still works on my end.

![screenshot101](https://github.com/mbebe/blomqvist/assets/5880037/bf7a494a-7804-432f-8576-37572226f2c3)

![screenshot103](https://github.com/mbebe/blomqvist/assets/5880037/10de9c47-a627-4bb7-a48c-0459c34ffdb7)

**Comment by NotTsuu** [comment]
> It still works on my end.
> 
> ![screenshot101](https://private-user-images.githubusercontent.com/5880037/325144159-bf7a494a-7804-432f-8576-37572226f2c3.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTM5NTY3OTAsIm5iZiI6MTcxMzk1NjQ5MCwicGF0aCI6Ii81ODgwMDM3LzMyNTE0NDE1OS1iZjdhNDk0YS03ODA0LTQzMmYtODU3Ni0zNzU3MjIyNmYyYzMucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDQyNCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA0MjRUMTEwMTMwWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9YjdmNzE1NWYyMWQ5ZTk4NDY4ZmM3NDU3YzliOWI4YmQ4ZmVhM2MyMDAzMzRhNjEzYmM4MGMxNjdiZWNjZjhlNSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.tw1WqtMgz8Gg87G6Zmt_PrydaNwpvoa23cFtCsR6wKg)
> 
> ![screenshot103](https://private-user-images.githubusercontent.com/5880037/325144219-10de9c47-a627-4bb7-a48c-0459c34ffdb7.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTM5NTY3OTAsIm5iZiI6MTcxMzk1NjQ5MCwicGF0aCI6Ii81ODgwMDM3LzMyNTE0NDIxOS0xMGRlOWM0Ny1hNjI3LTRiYjctYTQ4Yy0wNDU5YzM0ZmZkYjcucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDQyNCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA0MjRUMTEwMTMwWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NjRhNzU5OGMxOTRmM2I2NWNjNGJlNDQ4MGNlMzQ4YmRiYzRhOTg2OTFiOGE3ZDdjOGNkOThiN2ZjMDc0ZGI5MSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ._eZdDcdPYRWI4z9K_P-dKQSoEYrs9eaRK7XiSDIQXGo)

There are only a few episodes on the walking dead that say no links available. Even tho I checked the website and it was there.

**Comment by mkaflowski** [comment]
For me Viaplay doesnt work on Android Kodi Omega. On macos and previous Kodi version it works well.
